.github/workflows/deploy.yml

```diff
@@ -39,28 +39,32 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target:
-          - x86_64-unknown-linux-gnu
-          - x86_64-unknown-linux-musl
-          - i686-unknown-linux-musl
-          - x86_64-apple-darwin
-          - x86_64-pc-windows-msvc
         include:
           - target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
             name: starship-x86_64-unknown-linux-gnu.tar.gz
+            flags: "--features tls-vendored"
```

Comment:
I think every target can just use `--features tls-vendored` because openssl isn't used on windows and macos.

Reply:
Interesting I didn't know that, in that case would be better to make `tls-vendored` the default in the `Cargo.toml`?

Reply:
Not sure. It's not very relevant outside of getting distributable builds and it bloats the binary quite a bit. It would prevent issues with OpenSSL upgrades though.

Reply:
For now I've set it so that every target used the `tls-vendored` feature. We can look into moving this into `Cargo.toml` in the future if we need to.